mmv1/products/dialogflowcx/api.yaml

```diff
@@ -0,0 +1,115 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: DialogflowCX
+display_name: Dialogflow CX
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://{{location}}-dialogflow.googleapis.com/v3/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Dialogflow API
+    url: https://console.cloud.google.com/apis/library/dialogflow.googleapis.com
+objects:
+  - !ruby/object:Api::Resource
+    name: 'Agent'
+    base_url: "projects/{{project}}/locations/{{location}}/agents"
+    self_link: "{{name}}"
```

Comment:
```suggestion
```

Reply:
This should take on the value of `projects/{{project}}/locations/{{location}}/agents/{{name}}` automatically but if not, we'll want to set that explicitly.